CONTRIBUTING.md

```diff
@@ -1,143 +1,180 @@
+### Labels :label:
+Issues are marked with labels and some of them are not self-explanatory and are explained here:
+- **Good first issue**: :green_heart: Beginner friendly issues.
+- **Type-Labels**: Tells you what part of the architecture is involved. There are `type: Frontend`, `type: Print`, `type: Deployment` & `type: API` the architecture for those are partially documented in the [wiki](https://github.com/ecamp/ecamp3/wiki/architecture-frontend)
+- **Needs prototype**: :bulb: If you have an idea how to solve this issue: we'd like to see it. This issue needs a prototype before actual implementation begins since the specifications are somewhat vague. A prototype can be many things, whether your prototype is a sketch, mockup, partial implementation or something else is up to you.
+- **Feature request**: :rocket: An idea/request for a functionality, not ready to be implemented but ready to be discussed.
 
-### Git setup
+### :point_right: Starting with an issue
+To get started, find an issue that interests you. If you are new, we recommend selecting a [Good first issue](https://github.com/ecamp/ecamp3/labels/Good%20first%20issue).
+Please note that for other issues we recommend ones with the label [Ready for Implementation](https://github.com/ecamp/ecamp3/issues?q=is%3Aopen+is%3Aissue+label%3A%22Ready+for+implementation%22), which signifies that the issue should have clear definitions. If you have any questions, feel free to ask.
+If you are working on an issue, please leave a comment so that we can assign it to you, to make sure that the specifications are still up-to-date, and to prevent two people working on the same issue.
+Alternatively, open a draft pull request and mention the issue ID to signal that you are working on that particular issue.
+Please note that while the wiki can be helpful in understanding the project, it's not exhaustive (meaning there might be parts missing or out of date).
+If you have any questions, please comment on the issue for clarification or ask on Discord. We are happy to help and answer any questions you might have.
 
-We use a triangular git workflow. This means that all changes are first pushed to a contributor's fork of the repository, and then the changes are merged into the main fork via a pull request. In practice, setting up this workflow looks as follows:
+### Git setup :octocat:
+
+We use a triangular git workflow. This means that all changes are first pushed to a contributor's fork of the repository, and then the changes are merged into the main fork via a pull request.
+If you are an advanced git user you can set this up yourself.
+In practice, setting up this workflow looks as follows:
+
+1. Fork the main repository onto your GitHub account. To use the commands your configured git `user.name` should be exactly your git user name.
+   If you run the code below, and it outputs your GitHub username you are good to go.
+    ```shell
+    echo $(git config user.name)
```

Comment:
> Isn't the `echo $()` redundant? 

Yes
> Wouldn't it suffice to run `git config user.name`?

It would.

But also there is nothing to gain by removing it.

I wrote it that specific way because of line 41:
> If not you should replace the `$(git config user.name)` parts with your username [...]

This part was written for newer developers, it might help them to run a search and replace and it's easier to recognize that this (`$(git config user.name)`) part refers to the git username if it's always constant.


It doesn't have a huge impact eighter way but I think it brings a slight benefit. I see 0 (real) downsides.